plugins/inputs/ipmi_sensor/sample.conf

```diff
@@ -1,43 +1,56 @@
 # Read metrics from the bare metal servers via IPMI
 [[inputs.ipmi_sensor]]
-  ## optionally specify the path to the ipmitool executable
+  ## ipmitool binary path
+  ## Specify the path to the ipmitool executable
   # path = "/usr/bin/ipmitool"
-  ##
+
+  ## Use sudo
   ## Setting 'use_sudo' to true will make use of sudo to run ipmitool.
   ## Sudo must be configured to allow the telegraf user to run ipmitool
   ## without a password.
   # use_sudo = false
-  ##
-  ## optionally force session privilege level. Can be CALLBACK, USER, OPERATOR, ADMINISTRATOR
-  # privilege = "ADMINISTRATOR"
-  ##
-  ## optionally specify one or more servers via a url matching
+
+  ## Servers
+  ## Specify one or more servers via a url. If no servers are specified, local
+  ## machine sensor stats will be queried. Uses the format:
   ##  [username[:password]@][protocol[(address)]]
-  ##  e.g.
-  ##    root:passwd@lan(127.0.0.1)
-  ##
-  ## if no servers are specified, local machine sensor stats will be queried
-  ##
+  ##  e.g. root:passwd@lan(127.0.0.1)
   # servers = ["USERID:PASSW0RD@lan(192.168.1.1)"]
 
-  ## Recommended: use metric 'interval' that is a multiple of 'timeout' to avoid
-  ## gaps or overlap in pulled data
-  interval = "30s"
+  ## Session privilege level
+  ## Choose from: CALLBACK, USER, OPERATOR, ADMINISTRATOR
+  # privilege = "ADMINISTRATOR"
+
+  ## Interval
+  ## Interval time to collect metrics. This should be larger than the timeout
+  ## value to avoid gaps or overlap.
+  # interval = "10s"
+
+  ## Timeout
+  ## Timeout for the ipmitool command to complete.
+  # timeout = "20s"
 
-  ## Timeout for the ipmitool command to complete. Default is 20 seconds.
-  timeout = "20s"
+  ## Metric schema version
+  ## See the plugin readme for more information on schema versioning.
+  # metric_version = 1
 
-  ## Schema Version: (Optional, defaults to version 1)
-  metric_version = 2
+  ## Sensors to collect
+  ## Choose from:
+  ##   * sdr: default, collects sensor data records
+  ##   * chassis_power_status: collects the power status of the chassis
+  ##   * dcmi_power_reading: collects the power readings from the Data Center Management Interface
+  # sensors = ["sdr"]
 
+  ## Hex key
   ## Optionally provide the hex key for the IMPI connection.
   # hex_key = ""
 
+  ## Cache
   ## If ipmitool should use a cache
-  ## for me ipmitool runs about 2 to 10 times faster with cache enabled on HP G10 servers (when using ubuntu20.04)
-  ## the cache file may not work well for you if some sensors come up late
+  ## Using a cache can speed up collection times depending on your device.
   # use_cache = false
 
+  ## Cache path
```

Comment:
This does not add information as this is basically the name of the option. :-)

```suggestion
```

Reply:
Can you share why you do this in some places, but not others? Seems arbitrary when we add a header versus when we do not.

Reply:
Don't know, but I try to not put headers which do say the same as the option. I think if you do have groups of options it can make sense to put such a header but then I would expect "Cache" or "Cache settings" or similar.